Backend/index.js

```diff
@@ -101,6 +101,28 @@ app.get('/listings/user', async (req, res) => {
     }
 });
 
+app.get('/listings/:id', async (req, res) => {
+    const { id } = req.params;
+    
+    try {
+        const listing = await prisma.listing.findUnique({
+            where: { id: parseInt(id) },
+            include: {
+                seller: true,
+            }
+        });
+
+        if (!listing) {
+            return res.status(404).json({ message: 'Listing not found' });
+        }
+
+        res.status(200).json(listing);
```

Comment:
curious, how do you know which status number to return here? i saw in some cases it's 200 and some cases it's 201

Reply:
I mainly use 201 when i am doing post request because 201 returns that is has been created successfully. i use 200 for other ones because it returns "Everything is ok"